modules/light-clients/06-solomachine/misbehaviour_handle.go

```diff
@@ -41,7 +41,7 @@ func (cs ClientState) verifyMisbehaviour(cdc codec.BinaryCodec, misbehaviour *Mi
 // unmarshaled into the specified data type.
 func (cs ClientState) verifySignatureAndData(cdc codec.BinaryCodec, misbehaviour *Misbehaviour, sigAndData *SignatureAndData) error {
 	// do not check misbehaviour timestamp since we want to allow processing of past misbehaviour
-	if err := cdc.Unmarshal(sigAndData.Path, new(commitmenttypes.MerklePath)); err != nil {
+	if err := cdc.Unmarshal(sigAndData.Path, new(commitmenttypesv2.MerklePath)); err != nil {
```

Comment:
This was actually missed previously. But this is wire compatible so I don't see any issues. cc @colin-axner 

Plus it actually just throws away the merkle path regardless since its using `new()` inline, I'm not sure exactly what the point of this unmarshal is here tbh

Reply:
We actually have an [issue](https://github.com/cosmos/ibc-go/issues/4769) to change this, because indeed there is no really any need to proto-marshal the path.

Reply:
Aha, nice!